### PR TITLE
SW-4400 add colored progress for the leds

### PR DESCRIPTION
**Pull request by JFSolution**
update python
add mrb3 led strips

### CODE REVIEW

**Review by caglarkilimci — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -142,10 +142,13 @@ jobs:
         echo "Placed the image in ${target_dir}/${target_name}.bz2 with bmap file"
 
     - name: Build Image (MrBeam Variant)
+      env:
+        PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
Comment (irlaec) [inline .github/workflows/build.yml:146]: Is it necessary do add it here? I thought you could just use it directly, same as with `secrets.SSH_BEAMOS01_RSA`

File: src/modules/octopi/start_chroot_script [hunk 2]
```diff
@@ -34,21 +34,55 @@ else
 fi
 apt-get -y --force-yes install python3 python3-virtualenv git screen subversion cmake avahi-daemon libavahi-compat-libdnssd1 libatlas3-base
 
+#apt-get -y install software-properties-common
```
Comment (irlaec) [inline src/modules/octopi/start_chroot_script:37]: Please delete commented out code if we don't need it

Comment (JFSolution) [reply src/modules/octopi/start_chroot_script:37]: It is already merged 😅
I can remove it if i have a new PR

File: .github/workflows/build.yml (hunk 1 above)
Comment (JFSolution) [reply .github/workflows/build.yml:146]: It didnt work so I tried a lot 😅😅

File: src/modules/octopi/start_chroot_script (hunk 2 above)
Comment (irlaec) [reply src/modules/octopi/start_chroot_script:37]: Aaah I didn't notice :rofl:

**Pull request merged**